module/wa/module.py

```diff
@@ -967,18 +968,44 @@ def import_wa(i):
             pname=None
             imported_params={}
 
-            for name, obj in inspect.getmembers(cs):
-                if name!='AndroidUxPerfWorkload' and inspect.isclass(obj):
-                    try:
-                        addr=getattr(cs, name)
-                        pp=addr.parameters
-                        pname=addr.package
-                    except Exception as e:
-                        pass
-
-                    if pp!=None:
-                        wa_name=name
-                        break
+            # Only examine classes
+            for name, obj in inspect.getmembers(cs, predicate=inspect.isclass):
```

Comment:
Why not use WA's extension loader  to get it to discover the workload classes?
https://github.com/ARM-software/workload-automation/blob/master/wlauto/commands/create.py#L279